rule.go

```diff
@@ -160,6 +163,15 @@ func (cc *Conn) DelRule(r *Rule) error {
 	return nil
 }
 
+// HandleResponse retrieves Handle in netlink response
+func (r *Rule) HandleResponse(msg netlink.Message) {
+	rule, err := ruleFromMsg(msg)
+
+	if err == nil {
+		r.Handle = rule.Handle
```

Comment:
In Go, the happy path should not be indented. Instead, return early from the function if `err` is non-nil